tests/virt_autotest/virt_utils.pm

```diff
@@ -317,21 +318,18 @@ sub generate_guest_asset_name {
       . lc(get_required_var('SYSTEM_ROLE')) . '_'
       . get_required_var('ARCH');
 
-    record_info('Guest asset info', "Guest asset name is : $composed_name");
-
     return $composed_name;
 }
 
 sub get_guest_disk_name_from_guest_xml {
     my $guest = shift;
 
     # Our automation only supports single guest disk
-    my $disk_from_xml = script_output("virsh dumpxml $guest | sed -n \'/disk/,/\\\/disk/p\' | grep 'source file=' | grep -v iso");
+    my $disk_from_xml = script_output("virsh dumpxml $guest | xmlstarlet sel -t -v //disk/source/\@file";
     record_info('Guest disk config from xml', "Guest $guest disk_from_xml is: $disk_from_xml.");
     $disk_from_xml =~ /file='(.*)'/;
     $disk_from_xml = $1;
```

Comment:
I think these two lines:
   $disk_from_xml =~ /file='(.*)'/;
    $disk_from_xml = $1;
can be removed

Reply:
oh, yes, it was missed, good catching.